app/components/documents/document-card.hbs

```diff
@@ -5,22 +5,40 @@
   data-test-document-card
 >
   <div class="vlc-document-card__summary">
-    <div class="auk-u-mr-3">
-      {{#if @isHighlighted}}
-        <Auk::Badge @skin="success" @icon="document-added" @size="large" />
-      {{else}}
-        <Auk::Badge @icon="document" @size="large" />
-      {{/if}}
-    </div>
     <div class="vlc-document-card__content">
       <Auk::Toolbar @auto={{true}} as |Toolbar|>
         <Toolbar.Group @position="left" as |Group|>
           <Group.Item>
-            <p data-test-document-card-type class="auk-overline">
-              {{#unless this.loadPieceRelatedData.isRunning}}
-                {{this.documentContainer.type.label}}
-              {{/unless}}
-            </p>
+            {{#if @isHighlighted}}
+              <Auk::Badge @skin="success" @icon="document-added" @size="large" />
+            {{else}}
+              <Auk::Badge @icon="document" @size="large" />
+            {{/if}}
+          </Group.Item>
+          <Group.Item class="auk-u-maximize-width">
+            <div class="auk-o-flex auk-o-flex--vertical-center auk-o-flex--spaced">
+              <span data-test-document-card-type class="auk-overline">
+                {{#unless this.loadPieceRelatedData.isRunning}}
+                  {{this.documentContainer.type.label}}
+                {{/unless}}
+              </span>
+            </div>
+            <h6 class="auk-h4" data-test-document-card-name-value>
+              {{! template-lint-disable no-invalid-interactive }}
+              <span {{on "click" (fn this.showPieceViewer this.piece)}}>
```

Comment:
Do we want to keep this as a `<span>` with a bound action instead of a `<LinkTo>` (I'm guessing it's because of the styling, i.e. lack of underline)? Maybe instead of disabling the lint here we can add `role="link"` here.

Just so I know how to do it in the other tickets :)

Reply:
It's copied like this from the usertest branch. @Dietr Is it ok if we add `role="link"` to this span?

Reply:
Should be a button as it's an action. Added some styling for it on https://github.com/kanselarij-vlaanderen/frontend-kaleidos/commit/d3f85609dc104a6b45c8b50c434e408140cda1a1